content/en/news/releases/1.17.x/announcing-1.17/upgrade-notes/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Istio 1.17 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.17.0.
```

Comment:
They probably aren't going to all upgrade to 1.17.0 so make it generic to 1.17:
```suggestion
description: Important changes to consider when upgrading to Istio 1.17.
```

Reply:
Done.